### DISCUSSION TITLE
封面指导教师职称间距问题

### DISCUSSION THREAD

**Discussion by zhangjiegnxx** [tuna/thuthesis#879] [General]
thuthesis-v7.4.0的封面指导教师职称比较长的时候，间距看起来很奇怪
![捕获](https://github.com/tuna/thuthesis/assets/59642311/865d4224-f788-46dc-9265-089b09097544)

**Comment by zepinglee** [comment]
应该是参照“01-硕-1-中文5专业学位硕士封面（有专业领域）”这个封面吧？

<img width="661" alt="Screenshot 2023-05-15 at 22 33 17" src="https://github.com/tuna/thuthesis/assets/12290822/a3842d6f-dee3-4328-97dd-509daa36c6b8">

我好像错误地设成固定字间距 16bp 了。稍后修改一下。

https://github.com/tuna/thuthesis/blob/6121f55aa2a2c4b1165b719239c0d231c44a7a48/thuthesis.dtx#L4672-L4675

**Reply by zepinglee** [reply]
<img width="528" alt="Screenshot 2023-05-15 at 22 44 45" src="https://github.com/tuna/thuthesis/assets/12290822/2e7ff7de-7b5d-40a9-b115-c308bdc74022">

@zhangjiegnxx 这样（青色）应该没问题吧？

**Reply by zhangjiegnxx** [reply]
可以的，感谢~

**Reply by zepinglee** [reply]
问题来了，“01-硕-1-中文1学术学位硕士封面.doc”的职称是字号 16bp，字间距 6bp。如果遇到 5 个字的情况该怎么设置？

<img width="618" alt="Screenshot 2023-05-15 at 22 52 59" src="https://github.com/tuna/thuthesis/assets/12290822/277fc5d5-1182-496b-b1f5-735206a8a6b8">

再次吐槽一下，这每个封面的格式都不一样。

**Reply by zepinglee** [reply]
“01-硕-1-中文1学术学位硕士封面.doc” 改为，如果职称为三个字以上，则字间距为 0bp（青色）。

<img width="499" alt="Screenshot 2023-05-15 at 23 02 30" src="https://github.com/tuna/thuthesis/assets/12290822/dcec8c55-d1f8-43fc-96b3-72153cb32128">

**Reply by zepinglee** [reply]
在 c359034 中修复。
